cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_angular_service_based_app/templates.xml

```diff
@@ -7,6 +7,8 @@
     <templateExtension ref="api.ts" mergeStrategy="tsmerge"/>
     <templateExtension ref="environment.ts" mergeStrategy="tsmerge_override"/>
     <templateExtension ref="environment.prod.ts" mergeStrategy="tsmerge_override"/>
+    <templateExtension ref="model.ts" mergeStrategy="tsmerge"/>
+    <templateExtension ref="${variables.etoName#lower_case}.ts" mergeStrategy="tsmerge"/>    
```

Comment:
Do we need to define these templates? I thought it was just enough to use the external increment.

Can you try to remove them? I would like to know what happens.